classes/class-wc-connect-nux.php

```diff
@@ -467,38 +413,6 @@ public function show_banner_after_connection() {
 			) );
 		}
 
-		public function show_tos_banner() {
-			if ( isset( $_GET['wcs-nux-tos'] ) && 'accept' === $_GET['wcs-nux-tos'] ) {
-				WC_Connect_Options::update_option( 'tos_accepted', true );
-
-				$this->tracks->opted_in( 'tos_banner' );
-
-				wp_safe_redirect( remove_query_arg( 'wcs-nux-tos' ) );
-				exit;
-			}
-
-			$this->show_nux_banner( array(
-				'title'          => __( 'Almost ready to enjoy discounted shipping rates', 'woocommerce-services' ),
```

Comment:
We need to remember to regenerate the translation files before every release. Don't you have a release checklist somewhere, @jeffstieler ? We should put the checklist into a Wiki page.

Reply:
@jeffstieler shared p7bbVw-16e-p2 as a reference, with just 1-2 steps out of date IIRC.